tests/app/delivery/test_send_to_providers.py

```diff
@@ -730,3 +731,53 @@ def test_send_email_to_provider_uses_reply_to_from_notification(
         html_body=ANY,
         reply_to_address="test@test.com"
     )
+
+
+def test_send_sms_to_provider_should_return_template_if_found_in_redis(
+        mocker, client, sample_template
+):
+
+    from app.schemas import template_schema
+    template_dict = template_schema.dump(sample_template).data
+
+    notification = create_notification(template=sample_template,
+                                       to_field= '+447700900855',
+                                       normalised_to=validate_and_format_phone_number('+447700900855'))
+    mocker.patch(
+        'app.redis_store.get',
+        side_effect=[
+            json.dumps({'data': template_dict}).encode('utf-8'),
+        ],
+    )
+    mock_get_template = mocker.patch(
+        'app.dao.templates_dao.dao_get_template_by_id_and_service_id'
```

Comment:
Minor: initially I struggled to understand the relevance of this patch, until I looked at the definition of `SerialisedTemplate.get_dict`. This is coupled to it. We could simplify this test by just checking the call to the serialised model, and trust that it implements it efficiently (or have a separate test for the method itself).